include/swift/AST/Stmt.h

```diff
@@ -920,6 +922,45 @@ class CaseLabelItem {
   }
 };
 
+/// FallthroughStmt - The keyword "fallthrough".
+class FallthroughStmt : public Stmt {
```

Comment:
There are no changes here to FallthroughStmt. I just ran into issues with my uses for it in CaseStmt and then discovered that I couldn't just use a forward declaration of it = /.

Reply:
*So I moved it above CaseStmt.